packages/auth/lib/index.d.ts

```diff
@@ -1228,6 +1238,20 @@ export namespace FirebaseAuthTypes {
      * > It is recommended to use {@link auth#onAuthStateChanged} to track whether the user is currently signed in.
      */
     currentUser: User | null;
+
+    /**
+     * Sets the tenant id.
+     *
+     * #### Example
+     *
+     * ```js
+     * await firebase.auth().setTenantId('tenant-123');
+     * ```
+     *
+     * @param tenantId the tenantID current app bind to.
+     */
+    setTenantId(tenantId: string | null): Promise<void>;
```

Comment:
But in the test you wrote, it throws an error if null is sent in? Also the native methods do not take promises, they are synchronous, so I think it should be this (takes string only, returns void)?

```suggestion
    setTenantId(tenantId: string): void;
```